code/game/machinery/doors/airlock.dm

```diff
@@ -72,6 +72,7 @@
 	var/hasShocked = 0 //Prevents multiple shocks from happening
 	autoclose = 1
 	var/obj/item/device/doorCharge/charge = null //If applied, causes an explosion upon opening the door
+	var/obj/item/weapon/note //Any papers pinned to the airlock
```

Comment:
Just like your FACE!
On a more serious note, it's either that I make it look like the more ambiguous white sheet of paper, or I add thirteen _more_ pictures for around seven different airlock types, which seems like something incredibly minor when we already have a lot of missing or incomplete sprites in-game.